config/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+
+// @see https://docs.newrelic.com/docs/agents/nodejs-agent/installation-configuration
+require('newrelic');
```

Comment:
it was there initially, just thought it was better to move it out as part of the setup for the entry point. I'll put it back then.